controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/ConvergenceSummary.java

```diff
@@ -0,0 +1,130 @@
+// Copyright 2020 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.controller.deployment;
+
+import java.util.Objects;
+
+/**
+ * Summary of node and service status during a deployment job.
+ *
+ * @author jonmv
+ */
+public class ConvergenceSummary {
+
+    private final long nodes;
+    private final long down;
+    private final long upgradingOs;
+    private final long needPlatformUpgrade;
+    private final long upgradingPlatform;
+    private final long needReboot;
+    private final long rebooting;
+    private final long needRestart;
+    private final long restarting;
+    private final long services;
+    private final long needNewConfig;
+
+    public ConvergenceSummary(long nodes, long down, long upgradingOs, long needPlatformUpgrade, long upgradingPlatform,
+                              long needReboot, long rebooting, long needRestart, long restarting, long services, long needNewConfig) {
+        this.nodes = nodes;
+        this.down = down;
+        this.upgradingOs = upgradingOs;
+        this.needPlatformUpgrade = needPlatformUpgrade;
+        this.upgradingPlatform = upgradingPlatform;
+        this.needReboot = needReboot;
+        this.rebooting = rebooting;
+        this.needRestart = needRestart;
+        this.restarting = restarting;
+        this.services = services;
+        this.needNewConfig = needNewConfig;
+    }
+
+    /** Number of nodes in the application. */
+    public long nodes() {
+        return nodes;
+    }
+
+    /** Number of nodes allowed to be down. */
+    public long down() {
+        return down;
+    }
+
+    /** Number of nodes down for OS upgrade. */
+    public long upgradingOs() {
+        return upgradingOs;
+    }
+
+    /** Number of nodes in need of a platform upgrade. */
+    public long needPlatformUpgrade() {
+        return needPlatformUpgrade;
+    }
+
+    /** Number of nodes down for platform upgrade. */
```

Comment:
"Number of suspended nodes with pending platform upgrade"? E.g. we actually don't know whether a node is suspended because it plans to upgrade.